src/main/java/net/backupcup/mcde/MixinExtraInit.java

```diff
@@ -0,0 +1,11 @@
+package net.backupcup.mcde;
+
+import com.llamalad7.mixinextras.MixinExtrasBootstrap;
+
+import net.fabricmc.loader.api.entrypoint.PreLaunchEntrypoint;
+
+public class MixinExtraInit implements PreLaunchEntrypoint {
+    public void onPreLaunch() {
+        MixinExtrasBootstrap.init();
```

Comment:
Not correct way to initiate MixinExtras. See PR https://github.com/chronosacaria/MCDungeonsWeapons/pull/216 for explanation and https://github.com/chronosacaria/MCDungeonsWeapons/blob/6e46d46d9d9dee13262c820b830f5d4cc5df35f2/build.gradle#L72 for correct implementation

Reply:
I was using 0.1.1 release, so this wasn't necessary, although I switched to beta just in case for compatibility.